app/src/main/java/org/simple/clinic/summary/PatientSummaryScreen.kt

```diff
@@ -222,7 +222,7 @@ class PatientSummaryScreen(
 
```

Comment:
Can we rename the commit message to something like this

"Change tele-consult `ProgressMaterialButton` to `MaterialButton`"

in commit message you can mention about renaming `doctorButton` to `teleconsultButton`